src/pages/Pool/PoolPosition/PoolPosition.less

```diff
@@ -0,0 +1,52 @@
+.position__detail_wrapper {
+  width: 420px;
+  background-color: var(--ergo-pool-position-bg-1);
+}
+
+.percent-lbl.ant-typography {
+  padding: 2px 4px;
+  margin-left: 8px;
+  border-radius: 8px;
```

Comment:
Paddings and border-radius are `Box` responsibility
You can handle Margin using `Flex`